### PR TITLE
Initial CircleCI configuration

### PR DESCRIPTION
**Pull request by nemodjuric**
This is basic CircleCI configuration similar to what we had on Travis with better logging and additional failing test spec/features/backend_error_flows_spec.rb which is marked pending (see #1119 )
After this, we should introduce workflow similar to caseflow-certification with linting and security as parallel tasks.

### CODE REVIEW

File: ci-bin/capture-log [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+# capture-log
```
Comment (enriquemanuel) [inline ci-bin/capture-log]: What is this file for?

File: ci-bin/concatenate-log.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# frozen_string_literal: true
+
+# concatenate-log.rb
+
+require "json"
+require "open-uri"
+
+artifacts_url = "https://circleci.com/api/v1.1/project"\
+                "/github/department-of-veterans-affairs"\
+                "/caseflow/#{ENV['CIRCLE_BUILD_NUM']}/artifacts"
```
Comment (enriquemanuel) [inline ci-bin/concatenate-log.rb]: Is this right? This is reading from Caseflow not Efolder

**Comment by enriquemanuel** [comment]
@pkarman can you also check please?
And also check if you can remove TravisCi integration?

**Comment by enriquemanuel** [comment]
Just removed TravisCI

File: ci-bin/capture-log (hunk 1 above)
Comment (nemodjuric) [reply ci-bin/capture-log]: We used this to store CircleCI output. We do similar in caseflow-certification repo.

File: ci-bin/concatenate-log.rb (hunk 2 above)
Comment (nemodjuric) [reply ci-bin/concatenate-log.rb]: It should be caseflow-efolder

**Pull request merged**